server/bin/gold/test-5.2.txt

```diff
@@ -356,28 +356,26 @@ Summary Statistics for Tar Balls on 1970-01-01 (external data only):
                     1.8 kB size
 
         By Server Origin:
-                         3 "ONE" pbench satellite server
-                         5 main pbench server
+                         8 main pbench server
 
         Controller Counts:
                          6 controllers
-                         3 "ONE" pbench satellite server
-                         3 main pbench server
+                         6 main pbench server
 
 Tar Ball Counts broken down by weeks for most recent 2 months (with satellite percentages):
 
-Week Ending  Total Count      main       ONE
-1970-01-04 P           8    62.50%    37.50%
+Week Ending  Total Count      main
+1970-01-04 P           8   100.00%
 
 Tar Ball Counts broken down by Year (with satellite percentages):
 
-Year         Total Count      main       ONE
-1970       P           8    62.50%    37.50%
+Year         Total Count      main
+1970       P           8   100.00%
 
 Tar Ball Counts broken down by Month (with satellite percentages):
 
-Month        Total Count      main       ONE
-1970-01    P           8    62.50%    37.50%
+Month        Total Count      main
+1970-01    P           8   100.00%
```

Comment:
We are no longer getting the "satellite percentages" here...should we correct the summary header texts?  (Do we care?)

Reply:
I don't know after removing the prefix from the controller name how we will be able to distinguish between a dataset at audit level. I am not sure if we want to pursue this.

Reply:
See [above](https://github.com/distributed-system-analysis/pbench/pull/3187#discussion_r1082851464).